qiskit/pulse/commands/pulse_decorators.py

```diff
@@ -25,14 +25,14 @@ def function(func):
     Raises:
         CommandsError: when invalid function is specified.
     """
-    def to_pulse(duration, *args, name=None, **kwargs):
+    def to_pulse(duration, *args, **kwargs):
```

Comment:
We still want the user to be able to provide their own name for a pulse/command if they desire.